### ISSUE TITLE
Check for mp4 or other web embeddable content for asset embeds.

### ISSUE THREAD

**Issue by CodeFluent** [gastate/dalnfrontend#106]
Ben uploaded a m4v file to his post which was a video, but since m4v isn't web embeddable (compatible), the video doesn't show in the post detail view. 

This will be solved for checking the extension of the filename as well as making sure null sources in the post-asset component return non-null values.

**Comment by ShadSterling** [comment]
ffmpeg can convert.  (also validate, re-encode for maximum datarate or file size, etc, etc, etc)

**Comment by CodeFluent** [comment]
That would be excellent. I'll talk to @ShakibAhmed about it for a future enhancement. Perhaps docx files can be converted to pdf for better viewing for the web as well.

**Comment by ShadSterling** [comment]
@ShakibAhmed is supposed to be working on https://github.com/gastate/RCII, he's not the one to bring in for this.  If you need help getting started with ffmpeg I can point you in the right direction, but since this isn't really core functionality I don't want it to be a distraction.  If putting it behind core functionality means it'll be a while, in the short term you could add some instructions that tell submitters which video formats will be viewable in the browser.

**Comment by CodeFluent** [comment]
Absolutely, that's the plan. With Slalom soon coming aboard I hope they'll be able to talk to the clients to fit their needs. I think the main thing would be that all text uploads turn into pdf rather than video upload formats. The main work will be handled by Slalom at that point.

Added enhancement tag to this so it's not priority for now.

**Comment by ShadSterling** [comment]
"Closed" and "not a priority for now" are very different things.  It doesn't sound like this has been completed or cancelled, so I'm reopening.  If you want to be more specific about "for now" you can add a more specific tag or set a milestone.
